data/quarg/quarg news.txt

```diff
@@ -0,0 +1,425 @@
+			`"`
+news "engineer on quarg planets in human space"
+	location
+		government "Quarg"
+		attributes "human quarg"
+	name
+		word
+			"Engineer"
+			"Tarazed engineer"
+	message
+		word
+			`"`
+		word
+			"Quarg technology is so ahead of us we probably couldn't build their ships even if they gave us the plans."
+			"I'm working on a new spaceship for the Tarazed Corporation. Every once in a while I come here to take some inspiration from these magnificent designs."
+		word
+			`"`
+news "merchant on quarg planets in human and hai space"
+	location
+		government "Quarg"
+		attributes "human quarg" "hai quarg"
+	name
+		word
+			"Merchant"
+			"Captain"
+			"Ship captain"
+			"Merchant captain"
+			"Starship captain"
+			"Spacecraft captain"
+			"Spaceship captain"
+			"Freelance captain"
+			"Merchant pilot"
+			"Starship pilot"
+			"Spacecraft pilot"
+			"Spaceship pilot"
+			"Freelance pilot"
+	message
+		word
+			`"`
+		word
+			"The Quarg have impressive ships, but I bet my "
+		word
+			"Behemoth"
+			"Hauler III"
+			"Bulk Freighter"
+			"Bactrian"
+		word
+			" can carry twice the cargo their ships ever could."
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"The Quarg are an unwarlike species, but I've heard they fought battles against "
+		word
+			"an alien race"
+			"some insectoid aliens"
+		word
+			" "
+		word
+			"in the past"
+			"a long time ago"
+		word
+			"."
+		word
+			`"`
+news "merchant on quarg planets in human space"
+	location
+		government "Quarg"
+		attributes "human quarg"
+	name
+		word
+			"Merchant"
+			"Captain"
+			"Ship captain"
+			"Merchant captain"
+			"Starship captain"
+			"Spacecraft captain"
+			"Spaceship captain"
+			"Freelance captain"
+			"Merchant pilot"
+			"Starship pilot"
+			"Spacecraft pilot"
+			"Spaceship pilot"
+			"Freelance pilot"
+	message
+		word
+			`"`
+		word
+			"The Quargs go out of their way to keep piracy at bay in these systems. "
+		word
+			"I could fly a fleet of "
+		word
+			"Star Barges"
+			"Shuttles"
+			"Heavy Shuttles"
+		word
+			" around here!"
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"The Quarg are "
+		word
+			"peaceful"
+			"a peaceful race"
+			"not violent"
+		word
+			", "
+		word
+			"but I've seen their weapons at work: they "
+		word
+			"took out"
+			"disabled"
+			"destroyed"
+		word
+			" "
+		word
+			"a Leviathan pirate"
+			"a pirate ship"
+			"a maraduer ship"
+		word
+			" "
+		word
+			"in a matter of seconds"
+			"in a few seconds"
+			"without taking a single hit"
+		word
+			"."
+		word
+			`"`
+news "reporter on quarg planets in human space"
+	location
+		government "Quarg"
+		attributes "human quarg"
+	name
+		word
+			"Reporter"
+	message
+		word
+			`"`
+		word
+			"Read my new article:"
+			"Read my new article!"
+			"Read this!"
+			"Look at my new article!"
+			"Look at this!"
+		word
+			" '"
+		word
+			"Three"
+			"Five"
+			"Seven"
+		word
+			" "
+		word
+			"Things Tarazed Corporation Stole from the Quargs'"
```

Comment:
I'm pretty sure that the plural of 'Quarg' is 'Quarg'. Also, I would end these sentences with exclamation points, in order to keep with the 'street hawker' theme.
```suggestion
			"Things Tarazed Corporation Stole from the Quarg!"
```